cmd/main.go

```diff
@@ -63,7 +64,13 @@ func main() {
 		log.Fatalf("[error] : %v", err)
 	}
 
-	ctx := context.Background()
+	// kubelet default timeout for volumes is 2m3s - https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/volumemanager/volume_manager.go#L80
+	// setting the context to 1m50s will ensure request is terminated if taking longer/unable to establish connection due to underlying network error and
+	// the correct error is returned back to the driver
+	// Value is set to 1m50s to provide enough time for driver to complete outstanding operations
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute+50*time.Second)
```

Comment:
Should we make this configurable and default to 1m50s?

Reply:
@ritazh Good idea! We can add a `context-timeout` flag in the provider and default that to 1m50s for the time being. For addition of this flag as mandatory in the driver, I'll add an item in the agenda next week so we can discuss how we want to do this? This would require an update in all the providers to support the flag before we add to driver.